### PR TITLE
Adds nodemon and reload

### PR DESCRIPTION
**Pull request by Berkmann18**
This way, development/user tests can be done without having to manually
restart `npm start` and it also automatically reloads the web page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "Help slow the spread of COVID-19 by discouraging people from touching their face while sitting in front of a computer.",
   "main": "index.js",
   "scripts": {
-    "start": "node app.js"
+    "start": "node app.js",
+    "dev": "nodemon app"
```
Comment (misterpeddy) [inline package.json]: Yes this is fantastic!
Would you mind adding a quick line about it to README similar to the one on start script?

Comment (Berkmann18) [reply package.json]: Sure, I'll do that now.

**Comment by Berkmann18** [comment]
Hmm, it doesn't seem like the PR review lock is in place (yet) :thinking:.

**Comment by Berkmann18** [comment]
@AndyJiang99 There's definitely a way considering VueJS and React can do that although I'm not exactly sure how.

**Comment by AndyJiang99** [comment]
NPM has a reload package: https://www.npmjs.com/package/reload 
I'm not too sure if this can be implemented as we're writing in HTML code but it should be possible. If you have some time, take a quick swing at it but I'll also hop onto this tonight if this PR hasn't been closed yet.

**Comment by Berkmann18** [comment]
> NPM has a reload package: https://www.npmjs.com/package/reload
I've tried that following https://stackoverflow.com/a/55062832/5893085 but it didn't work, I wonder if it would work following the README of the package.

I've come across this (high-level) explanation on Webpack's Hot Module Replacement: https://stackoverflow.com/a/24587740/5893085

**Comment by Berkmann18** [comment]
It should be good to go.

File: README.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ One of the ways you can help the project is by collecting data to train our mode
 
 Clone the repo and run:
 
-`npm install`
+1. `npm install`
 
-`npm start`
+2. `npm start` or `npm run dev` (like `npm start` but the server automatically restarts after files are changed)
```
Comment (misterpeddy) [inline README.md]: re "files are changed" - is this any file? Or any `.js` file?

File: app.js [hunk 3]
```diff
@@ -14,5 +15,10 @@ app.use(MODELS_MOUNT_PATH, express.static(MODELS_DIR));
 
 app.get('/', (req, res) => res.sendFile(INDEX_PATH));
 
-// eslint-disable-next-line no-console
-app.listen(PORT, () => console.log(`Listening on port ${PORT}`));
+reload(app)
+  .then(() => {
+    // eslint-disable-next-line no-console
```
Comment (misterpeddy) [inline app.js]: Hmm is there a way of logging things to stdout/stderr that the linter does look favorably upon?

File: index.html [hunk 4]
```diff
@@ -74,5 +74,7 @@ <h2>Hands-Down!</h2>
         </div>
       </div>
     </div>
+
+    <script src="/reload/reload.js"></script>
```
Comment (misterpeddy) [inline index.html]: Oh this is cool! So when you import a package with npm, its source becomes available at the root directory to all scripts the runtime starts?

File: README.md (hunk 2 above)
Comment (Berkmann18) [reply README.md]: Ooh, good catch, I've specified which ones (i.e. any HTML/CSS/JS files).

File: app.js (hunk 3 above)
Comment (Berkmann18) [reply app.js]: Good point, I'll see what I can do to make ESLint happy without losing the logging ability.
We could use either `nclr`, `winstone`, `debug`, `signale` or others.
I'm not sure what the best approach would be for browser logging as I haven't used logging libraries there (other than using `console....()`).

File: index.html (hunk 4 above)
Comment (Berkmann18) [reply index.html]: Yeah, it's an interesting thing about node_modules (I don't know how that works tho).

**Comment by Berkmann18** [comment]
> Woohoo - thanks guys!
> Does this support hard reloading (no cached assets)?

Yes, I believe so.

File: app.js (hunk 3 above)
Comment (misterpeddy) [reply app.js]: sg! Not blocking this PR - we should just keep it in mind and adopt one of these that makes the most sense!

Comment (Berkmann18) [reply app.js]: Yeah, I've opted for `nclr` for the `app.js` which works wonders and don't require ESLint directives.

**Comment by Berkmann18** [comment]
@AndyJiang99 Does the hot-reloading works on your end?

**Comment by AndyJiang99** [comment]
> @AndyJiang99 Does the hot-reloading works on your end?

Yea just tested this on the master branch, reloads the server and the website! Thanks for the fix

**Pull request merged**